pkgs/applications/science/electronics/qcsxcad/default.nix

```
@@ -0,0 +1,52 @@
+{ stdenv
+, mkDerivation
+, fetchFromGitHub
+, cmake
+, csxcad
+, tinyxml
+, vtkWithQt5
+, qt5
+}:
+
+let
+  version = "0.6.2";
+in
+mkDerivation {
+  pname = "qcsxcad";
+  version = version;
```

I find this more readable
```suggestion
mkDerivation rec {
  pname = "qcsxcad";
  version = "0.6.2";
```